plugin/evm/mempool.go

```diff
@@ -145,13 +150,17 @@ func (m *Mempool) Add(tx *GossipAtomicTx) error {
 	return m.AddTx(tx.Tx)
 }
 
-// Add attempts to add [tx] to the mempool and returns an error if
-// it could not be addeed to the mempool.
+// AddTx attempts to add [tx] to the mempool and returns an error if
+// it could not be added to the mempool.
 func (m *Mempool) AddTx(tx *Tx) error {
 	m.lock.Lock()
 	defer m.lock.Unlock()
 
 	err := m.addTx(tx, false)
+	if errors.Is(err, errTxAlreadyKnown) {
+		return nil
+	}
```

Comment:
We call `addTx` in `AddLocalTx` and `ForceAddTx` - don't we need to check for this there as well?

Reply:
Yeah we do...